lib/galaxy/webapps/galaxy/controllers/history.py

```diff
@@ -664,7 +664,9 @@ def display_by_username_and_slug( self, trans, username, slug ):
         # Get history.
         session = trans.sa_session
         user = session.query( model.User ).filter_by( username=username ).first()
-        history = trans.sa_session.query( model.History ).filter_by( user=user, slug=slug, deleted=False ).first()
+        history = trans.sa_session.query( model.History ) \
+            .options( eagerload( 'tags' ) ).options( eagerload( 'annotations' ) ) \
```

Comment:
Is this different from the following? `.options( eagerload( 'tags' ), eagerload( 'annotations' ) )`